src/middleware/errors.js

```diff
@@ -0,0 +1,21 @@
+const { env } = require('../config');
+
+function logError(err) {
+  // TODO: config rollbar or something like that
+  if (!env.isTest) console.error(err); // eslint-disable-line no-console
```

Comment:
Same as before comment.
```suggestion
  if (env.isDev || env.isProd) console.error(err); // eslint-disable-line no-console
```

Reply:
I think you missed this one @fjsandov